src/app/server/Server.cpp

```diff
@@ -359,7 +359,10 @@ CHIP_ERROR Server::Init(const ServerInitParams & initParams)
                      chip::app::InteractionModelEngine::GetInstance());
 
     // Register Test Event Trigger Handler
-    mTestEventTriggerDelegate->AddHandler(&mICDManager);
+    if (mTestEventTriggerDelegate != nullptr)
```

Comment:
Please add a log warning that's it's null before these lines, in the earlier parts of Server::Init.